confidant_client/cli.py

```diff
@@ -457,7 +459,7 @@ class _HelpAction(argparse._HelpAction):
 
     def __call__(self, parser, namespace, values, option_string=None):
         parser.print_help()
-        print ''
+        print('')
```

Comment:
`print()` is equivalent to `print('')`

(also `print` is equivalent to `print ''` in python2)